test-e2e/build/custom-prefix.test.js

```diff
@@ -0,0 +1,18 @@
+const path = require('path');
+
+const {initFixture, esyCommands} = require('../test/helpers');
+
+it('Build - custom prefix (not propperly implemented)', async done => {
```

Comment:
why `(not propperly implemented)`?

Reply:
That should be removed, it is correct now